LICENSE

```diff
@@ -0,0 +1,340 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
What this license for?

Reply:
The flatpak manifest is copied from the upstream, the license of the upstream project is gpl-v2.

Reply:
Yes, please remove it.